composer.json

```diff
@@ -14,9 +14,9 @@
        },
     "require": {
         "php": "^8.0",
-        "lanfix/goaop-framework": "^4.0",
+        "sourcetoad/goaop-framework": "4.0.4",
```

Comment:
if you try to composer install this it doesn't work

Reply:
Also please run composer install after upping a dependency so that the composer.lock is updated and added to the commit.

Reply:
I got this to work, and here is my composer.json file.
<img width="1190" alt="Screenshot 2024-06-06 at 12 59 14 PM" src="https://github.com/sourcetoad/aspect-mock/assets/28880397/b58bc891-576e-4960-8caa-d0fca7b5879e">